src/WatchDogs.Application/Program.cs

```diff
@@ -1,14 +1,45 @@
+
+
+
+using Contracts;
+using Infrastructure.DxTrade;
+using System.Net.Http.Headers;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
+builder.Services.Configure<DxTradeConnectionOptions>(
+    builder.Configuration.GetSection(nameof(DxTradeConnectionOptions)));
 
 builder.Services.AddControllers();
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
+
+builder.Services.AddHttpClient("DxTradeAuthenticationClient", client => {
```

Comment:
Don't forget to replace the magic string with the constant

Reply:
Already done.

Reply:
DOne.